window.go

```diff
@@ -351,7 +361,19 @@ func (w *Window) SetSize(width, height int) error {
 func (w *Window) GetFramebufferSize() (width, height int, err error) {
 	var wi, h C.int
 	C.glfwGetFramebufferSize(w.data, &wi, &h)
-	return int(wi), int(h), fetchError()
+	return int(wi), int(h), fetchError() // FIXME: width and height are allowed to be NULL in C API
```

Comment:
I really don't know in which case GLFW will return null. I'm guessing errors? In that case we can merge it for now. This is the idiomatic way and I would like to avoid doing multiple returns the C way as long as we don't criple functionality.